art-Creating_a_Service_for_VM_Provisioning/topics/Provisioning_a_VM_Using_SSUI.adoc

```diff
@@ -0,0 +1,20 @@
+[[provision-vm-using-ssui]]
+== Provisioning a Virtual Machine Using the Self Service User Interface
+
+. Log in to the Self Service User Interface at https://<your_cloudforms_appliance>/self_service.
+. Click the *Service Catalog* tab.
+. Click the service you created.
+. In the *Service and VM Name*:
+.. Enter the service name in *Service Name*.
+.. Enter the virtual machine name in *VM Name*.
+.. Enter the virtual machine description in *VM Description*.
+. In *VM Characteristics*, enter *Number of CPUs* and *VM Memory*.
+. Click *Add to Shopping Cart* and access the shopping cart by clicking its icon on the upper-right corner of the screen.
+. Click *Order*. You have ordered a new provisioning request. You can follow its progress and see additional details of the order request by selecting it from the *Orders* tab.
+
+See the link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html-single/self_service_user_interface_guide/[Self Service User Interface Guide] for more information about using the Self Service user interface.
+
+
+.Summary
+
+As part of the above sample service, you created a provisioning dialog that defines the options that can be set on a virtual machine and you created a service dialog which allows us to expose certain options to be set by the user. For our example, the service name, virtual machine name, virtual machine description, and virtual machine characteristics are configurable. Then, you created a service catalog and finally a catalog item. The catalog item combines the service dialog with all of the options in the provisioning dialog. Lastly, you provisioned a Red Hat Virtualization virtual machine using the Self Service user interface.
```

Comment:
This first sentence can be broken out into two sentences. "Then, you created a service dialog...." Once you discuss the example, I would revise the next sentence to read something like, "Additionally, you created a service dialog..."